e2e/e2etests/test_solana_deposit_and_call_revert.go

```diff
@@ -24,12 +27,26 @@ func TestSolanaDepositAndCallRevert(r *runner.E2ERunner, args []string) {
 
 	// execute the deposit transaction
 	data := []byte("hello reverter")
-	sig := r.SOLDepositAndCall(nil, reverterAddr, depositAmount, data)
+	revertAddressPrivateKey, err := solana.NewRandomPrivateKey()
```

Comment:
NIT : maybe a revert option with incorrect address would be easy to add . 
Up to you , its more like good to have and not needed